app/presenters/finder_presenter.rb

```diff
@@ -140,6 +145,9 @@ def sort_options
     return [] if sort.blank?
 
     options = sort.collect do |option|
+      # FIXME: prototype only
+      option['name'] = 'Topic' if option['name'] == 'Most relevant'
```

Comment:
Does this still require refactoring or can the FIXME be deleted?

Reply:
This should come from the finder content item sort options.

Reply:
Removed in https://github.com/alphagov/finder-frontend/pull/856/commits/97fb0e9cfc1090633780c923c9186c6ab6758640 and present in the finder now see: https://github.com/alphagov/govuk-app-deployment-secrets/pull/61/commits/9e1cbc970cb67f6e2405d47d1c2074e32438a6d2